frontend/src/Feed.js

```diff
@@ -48,8 +49,24 @@ class Feed extends React.Component {
     // Additional documentation about React's lifecycle can be found here:
     // https://reactjs.org/docs/react-component.html#commonly-used-lifecycle-methods
     componentDidMount() {
-        // Make a call to our API
-        fetch('/feed')
+       this.getMessages();
+    }
+
+    onTextChange(event) {
+        event.preventDefault();
+        this.setState({text: event.target.value});
+    }
+
+    onButtonClick(event) {
+        event.preventDefault();
+        fetch(`/messages?text=${this.state.text}`, {method: 'POST'})
+            .then((response) => this.getMessages())
```

Comment:
If the response completes successfully we'll fetch the messages again by calling getMessages()(the latest one will be included).